src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] @Moise-code it's not best practice to disable the linter check in a file, kindly fix the linter errors instead of disabling them.